src/main/kotlin/autolotto/domain/Amount.kt

```diff
@@ -0,0 +1,17 @@
+package autolotto.domain
+
+class Amount(val amount: Int) {
+    init {
+        require(amount > PURCHASE_AMOUNT_NONE_MATCH) { "금액은 0보다 커야합니다." }
+        require(amount >= LOTTO_AMOUNT && amount % LOTTO_AMOUNT == PURCHASE_AMOUNT_NONE_MATCH) { "돈은 ${LOTTO_AMOUNT}원 단위로만 입력 가능합니다." }
+    }
+
+    fun getLottoGameCount(): Int {
+        return amount / LOTTO_AMOUNT
+    }
```

Comment:
읽기 전용 프로퍼티로 나타내 보아도 좋을것 같습니다.

ex
```suggestion
    val lottoGameCount: Int
        get() = amount / LOTTO_AMOUNT
```